### ISSUE TITLE
Virtual Batch Normalization using Tensorflow

### ISSUE THREAD

**Issue by daanklijn** [tensorflow/tensorflow#46572]
What is the recommended way to use [VBN](https://arxiv.org/pdf/1606.03498v1.pdf) using Tensorflow? 
Would it be an idea to add this to the documentation?

**Comment by omalleyt12** [comment]
You can use the `virtual_batch_size` argument in `BatchNormalization` to do this. It's undocumented bc when it was exposed, we didn't see many users of it
